src/ebmc/transition_system.cpp

```diff
@@ -221,6 +222,10 @@ int get_transition_system(
   if(cmdline.isset('I'))
     config.verilog.include_paths = cmdline.get_values('I');
 
+  // do --system-verilog
+  if(cmdline.isset("system-verilog"))
+    verilog_languaget::force_systemverilog = true;
```

Comment:
We have `languaget::set_language_options`, which seemingly isn't used in the `hw-cbmc` code base (while the cbmc code base does use it). So I think this global variable can be avoided by instead putting information in `options` and making sure that `set_language_options` is called.